subsys/bluetooth/audio/bap_broadcast_source.c

```diff
@@ -523,8 +523,8 @@ static bool valid_create_param(const struct bt_bap_broadcast_source_create_param
 		return false;
 	}
 
-	CHECKIF(param->params_count == 0U) {
-		LOG_DBG("param->params_count is 0");
+	CHECKIF(!IN_RANGE(param->params_count, 1, CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT)) {
```

Comment:
nitpick 
```suggestion
	CHECKIF(!IN_RANGE(param->params_count, 1U, CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT)) {
```

Reply:
I'll update that - This one is kind of interesting though, because `CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT` is still signed int :D

Reply:
yes, I noticed that but other instances had `U` in the same file, I let you take the call.

Reply:
It's a question of whether comparing a unsigned value with 2 signed values, or compare an unsigned value with another unsigned value and an unsigned value with an signed value. 

I'd say that the latter, i.e. the one with most "signed correctness" is the best :)